### DISCUSSION TITLE
Can't integrate with Home Assistant

### DISCUSSION THREAD

**Discussion by nunmen** [HASwitchPlate/openHASP#190] [Q&A] [answered]
Hi openHASP.
Coongratulations for this great project.

I've followed the manual [How-To](https://haswitchplate.github.io/openHASP-docs/0.6/integrations/home-assistant/howto/) installation of openHASP in Home assistant.
I've configured the MQTT server.
I've added the pages.jsonl to the plate and the openhasp and plate to homeassistant.

The plate publishes in the MQTT server but the integration dont apear.
If I try to add the integration manualy I receive the message "Configuration is done through discovery".

HomeAssistant log:
`WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration openhasp which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant`

What can I be doing wrong?

Thanks

**Comment by nagyrobi** [comment, marked as answer]
That's a universal message displayed in the log for each _custom integration_ installed in Home Assistant. It's safe to ignore it.

Go to integrations menu, you should have your plates discovered waiting to be pre-configured.

**Reply by nunmen** [reply]
Thanks nagyrobi for the fast repply.

They aren't there: 
![image](https://user-images.githubusercontent.com/87753197/126915380-69113838-6fd0-4440-9c6b-74cfb2db4455.png)

**Reply by nagyrobi** [reply]
Reboot both plates and HA.

**Reply by nunmen** [reply]
Rebooted both.

The MQTT server gets the message `hasp/discovery {"node":"plate_sala","mdl":"esp32dev-ili9488","mf":"openHASP","hwid":"30aea...","pages":12,"sw":"0.6.1","input":{},"power":[],"light":[],"dim":[]}` but HomeAssistant simply ignores it.

I can't find anything in the homeassistant logs for openhasp.

Edit:
I uninstalled the manualy installed version, installed HACS and installed openHASP trough HACS.
Had the same result.
The plate didn't appear in the integrations page.

**Reply by nunmen** [reply]
Update:

I added ` custom_components.openhasp: info` to HomeAssistant logs.
Openhasp don't write any logs, even if I force an error.

Is there any issue with openhasp and the last version of Homeassistant?

**Reply by nagyrobi** [reply]
Try with `debug` instead of  `info`.

**Reply by nunmen** [reply]
With debug instead of info, I get the same.
No logs and no integrations.

**Reply by nagyrobi** [reply]
There is some problem with your HA instance, we can't tell what.

**Reply by nunmen** [reply]
Installed and reconfigured Home Assistant from scratch.
Now is all working fine.

Congratulations.
**OpenHASP is a great piece of software!**

**Comment by Alecs2000** [comment]
у меня такая же проблема Configuration is done through discovery
